docs/overview/storage_consistency.txt

```diff
@@ -134,6 +134,16 @@ always be visible, since the information will be retrieved directly from the
 translog.
 
 .. NOTE::
+  
+  ``Dirty reads`` can occur if the primary shard becomes isolated. The primary 
+  will only realize it is isolated once it tries to communicate with its 
+  replicas or the master. At that point, a write operation is already committed 
+  into the primary and can be read by a concurrent read operation. In order to 
+  minimise the window of opportunity for this phenomena, the CrateDB nodes 
+  communicate with the master every second (by default) and once they realise 
```

Comment:
"once it realizes"

Reply:
The subject is "the CrateDB nodes" (collective). In order to use "it" and singular (ie. it realizes and it will) should we change "the CrateDB nodes communicate with the master" to just "CrateDB communicates with the master" ? (the nodes communicate with the master as well as the master with the nodes. In this paragraph it's one of the nodes that realizes it's disconnected)

Reply:
@nslater ^^

Reply:
my mistake. I missed the plural. ignore this comment! ty